tests/overload.erl

```diff
@@ -107,14 +103,8 @@ test_vnode_protection(Nodes, BKV, ConsistentType) ->
     rt:pmap(fun(Node) ->
                     rt:update_app_config(Node, Config)
             end, Nodes),
-    ProcFun = fun(X) ->
-                      lager:info("in test_vnode_protection ProcFun, Procs:~p, Metric:~p", [X, (2*?NUM_REQUESTS * 1.5)]),
-                      X =< (2*?THRESHOLD * 1.5)
-              end,
-    QueueFun = fun(X) ->
-                      lager:info("in test_vnode_protection QueueFun, QueueSize:~p, Metric:~p", [X, (?NUM_REQUESTS * 1.1)]),
-                      X =< (?THRESHOLD * 1.1)
-              end,
+    ProcFun = build_predicate_lt(test_vnode_protection, (2*?THRESHOLD * 1.5), "ProcFun", "Procs"),
```

Comment:
I don't really understand why # of procs wouldn't always be equal to ?NUM_REQUESTS. I probably just don't understand how overload_proxy actually fits in here.